wagtailio/core/blocks.py

```diff
@@ -245,7 +264,7 @@ class Meta:
         label = "Card"
 
 
-class LogoCardBlock(blocks.StructBlock):
+class LogoCardBlock(CTALinkMixin, blocks.StructBlock):
```

Comment:
Note: We do not need the second inherited model.

Reply:
Fixed in 9d75fc4